resource-server/okta-aspnetcore-webapi-example/Controllers/MessageController.cs

```diff
@@ -24,12 +25,12 @@ public JsonResult Get()
                 .SingleOrDefault(c => c.Type == ClaimTypes.NameIdentifier)
                 ?.Value;
 
-            return Json(new
+            return new JsonResult(new
             {
                 messages = new dynamic[]
                 {
-                    new { Date = DateTime.Now, Text = "I am a Robot." },
-                    new { Date = DateTime.Now, Text = "Hello, world!" },
+                new { Date = DateTime.Now, Text = "I am a Robot." },
```

Comment:
Indentation (although not entirely sure in this instance of dynamic object initializer)